### ISSUE TITLE
'no photos' message shows up even when there are photos in the gallery

### ISSUE THREAD

**Issue by gitaaron** [functionland/fula#211]
If I connect the sample gallery to a Box and throttle the connection.  After logging in I see the 'no photos' message even if I have already added photos to the gallery.  Once the collection loads then the message disappears and the photos are displayed.

**Comment by ehsan6sha** [comment]
@gitaaron does it mean the message needs to be changed to something like loading or there is a root error here?

**Comment by gitaaron** [comment]
I think changing it to something like 'loading' is good no root error in
the protocol maybe just a fix in the sample?  @faroud correct me if I'm
wrong

**Comment by farhoud** [comment]
> I think changing it to something like 'loading' is good no root error in the protocol maybe just a fix in the sample? @Faroud correct me if I'm wrong

You are right the code in `src/components/Gallery.jsx` look only if the photos state and if the size is 0 show the no photos.

**Comment by farhoud** [comment]
> I think changing it to something like 'loading' is good no root error in the protocol maybe just a fix in the sample? @Faroud correct me if I'm wrong.

your are right,
